src/javascript/app/pages/user/metatrader/metatrader.config.js

```diff
@@ -153,12 +168,19 @@ const MetaTraderConfig = (() => {
 
                     const has_financial_account = Client.hasAccountType('financial', 1);
                     const is_maltainvest        = State.getResponse(`landing_company.mt_financial_company.${getMTFinancialAccountType(acc_type)}.shortcode`) === 'maltainvest';
-                    const is_financial          = accounts_info[acc_type].account_type === 'financial';
                     const is_demo_financial     = accounts_info[acc_type].account_type === 'demo' && accounts_info[acc_type].mt5_account_type; // is not demo vol account
+                    const is_financial          = accounts_info[acc_type].account_type === 'financial';
 
                     if (is_maltainvest && (is_financial || is_demo_financial) && !has_financial_account) {
                         $message.find('.maltainvest').setVisibility(1);
-                        resolveWithMessage();
+                        
+                        BinarySocket.wait('get_account_status').then((response) => {
+                            if (Client.get('residence') === 'gb' && !/age_verification/.test(response.get_account_status.status)) {
+                                $message.find('.authenticate').setVisibility(1);
+                            }
+
+                            resolveWithMessage();
+                        });
```

Comment:
why don't we wrap `Client.get('residence') === 'gb'` around this too so we don't wait for get account status if it's not needed?

Reply:
fixed